admin_guide/sysctls.adoc

```diff
@@ -135,31 +136,49 @@ ifdef::openshift-origin[]
 endif::[]
 
 [[setting-sysctls-for-a-pod]]
-== Setting Sysctls for a Pod
+== Setting sysctls for a pod
```

Comment:
@kalexand-rh This heading appears to be formatted incorrectly on the View page here. But, I think it is OK; the formatting looks off in all points in the repo history since the ifdefs were added right before the heading.

Reply:
Interesting. It looks ok in a local build, too. Thanks for checking it.